library/globals.inc.php

```diff
@@ -2955,7 +2955,7 @@ function gblTimeZones()
             '0',
             xl('Rx Show State Lic. #')
         ),
-        'rx_show_drug-drug' => array(
+        'rx_show_drug_drug' => array(
```

Comment:
btw, I had to do this since the dash was not being brought into the globals smarty array. Also, this global was not being used anywhere anyways, so ok to change it.